test/extended/cli/apiresources.go

```diff
@@ -41,6 +38,27 @@ var _ = g.Describe("[sig-cli] oc api-resources", func() {
 		o.Expect(out).To(o.ContainSubstring("serviceaccounts"))
 		o.Expect(out).To(o.ContainSubstring("deployments"))
 
+		out, err = oc.Run("api-versions").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("apps/v1"))
+		o.Expect(out).To(o.ContainSubstring("networking.k8s.io/v1"))
+		o.Expect(out).To(o.ContainSubstring("node.k8s.io/v1"))
+		o.Expect(out).To(o.ContainSubstring("storage.k8s.io/v1"))
+	})
+
+	g.It("can output expected information about openshift api-resources [apigroup:build.openshift.io][apigroup:project.openshift.io][apigroup:user.openshift.io]", func() {
```

Comment:
I'm not sure about this part of the change. Logically it's saying that if the API exists then it should show up in the output from api-resources, but aren't we using the same query to see if it exists that oc uses to produce its output? Is that a useful test?

I know we've said we didn't want to generally, but this feels like a case where we _should_ say "If the cluster is MicroShift, skip" so that we can have a separate test that checks for the expected APIs when the cluster is MicroShift.